app/frontend/entrypoints/application.js

```diff
@@ -39,15 +39,4 @@ document
 
 initAll()
 
-// We set rawAttribute to true to pass the value of the text input to the
-// controller when the autocomplete is being used, so we can detect when the
-// user has cleared the field. We set source to false to overide the DfE
-// component's handling of sort order and synonyms, as described in the code
-// found in
-// https://github.com/DFE-Digital/dfe-autocomplete/blob/36d80e6b5bba67c92cd9ec6982a4e536d1889aed/src/dfe-autocomplete.js#L54C1-L54C1
-// this means the autocomplete will get options straight from the select
-// element, removing the null option, which isn't needed for autocomplete.
-dfeAutocomplete({
-  rawAttribute: true,
-  source: false
-})
+window.dfeAutocomplete = dfeAutocomplete
```

Comment:
put the dfeAutocomplete function into the window object. This allows us to call it with different options from inline scripts in views.

This breaks encapsulation but I don't think there is a better way.

Reply:
Is it worth raising a feature request at https://github.com/DFE-Digital/dfe-autocomplete/ asking for the ability to configure the component with `data-` attributes or something similar?

Reply:
Or just raising the problem of `setupAccessibleAutocomplete` not being exported as an issue, like you said in the PR comment and @DavidBiddle said 😅

Reply:
yeah - I think data- attributes are the way to go but probably a bit much to ask for :) I'll stick a PR in as you both suggest